helm/robusta/values.yaml

```diff
@@ -190,12 +192,25 @@ builtinPlaybooks:
   - job_events_enricher: {}
   - job_pod_enricher: {}
 
+- triggers:
+  - on_prometheus_alert:
+      alert_name: KubeletTooManyPods
+  actions:
+  - node_pods_capacity_enricher: {}
+  - alert_explanation_enricher:
+      alert_explanation: "You are approaching the limit of how many pods you can run on your node. This in and of its self is not an error but may impact the future scheduling of pods on this node."
```

Comment:
test suggestion:
explanation: "The node is approaching the maximum number of scheduled pods"
resolution: "Verify that you defined proper resource requests for your workloads. If pods cannot be scheduled, add more nodes to your cluster."

Reply:
added